### DISCUSSION TITLE
New YT Video about ZT-5566SE

### DISCUSSION THREAD

**Discussion by ludwich66** [webspiderteam/Bluetooth-DMM-For-Windows#35] [Show and tell]
"Darren Walker, ZT-5566SE Bluetooth CHEAP-O Multimeter Review & Teardown!":https://www.youtube.com/watch?v=hUhaA0PQmd8

**Comment by webspiderteam** [comment]
Ohh nice :) that's what we're looking for ☺️ did you try to contact?

**Reply by webspiderteam** [reply]
I wrote a comment under video. I hope we can get a response.

**Reply by ludwich66** [reply]
+++
i changed Read.me file

Happy New Year!!
Ludwich

**Reply by webspiderteam** [reply]
Thank you 😊 happy new year too

**Reply by ludwich66** [reply]
Hi Engin,
did you see Darrens comment?

<img width="939" alt="image" src="https://user-images.githubusercontent.com/12202733/211201671-d6d65f75-b113-42dd-a7c0-27b0fe42cb44.png">

**Reply by webspiderteam** [reply]
Yes I saw and also sent a mail same day. And he responded that he will test it and return. But he didn't send anything yet.

**Comment by kozakserwis** [comment]
Hello. My ZOYI ZT-5566SE does not show results on the computer but connects to the program via bluetooth correctly. How can I send logs?
Tested on Windows 11

**Reply by webspiderteam** [reply]
You can add files on comments here. Also you can send via email to webspiderteam@gmail.com

**Reply by kozakserwis** [reply]
> You can add files on comments here. Also you can send via email to [webspiderteam@gmail.com](mailto:webspiderteam@gmail.com)

can you tell me what files? Do I need a special program to generate logs?

**Reply by webspiderteam** [reply]
No you don't need any other program. Log files are under where you install program and named like log-2023-01-15.txt . It logs day by day if you send all logs that's enough. But before sending connect to device and do some measurements for getting different data makes results more correct. Like ohm capacitance voltage e.g.

**Reply by kozakserwis** [reply]
I dont have any logs
See on video:
https://www.youtube.com/watch?v=wzjoKyAVBvE

**Reply by webspiderteam** [reply]
its strange. it looks like gets no data from device.

**Reply by webspiderteam** [reply]
can you try this? 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424586/Portable.zip)

this version logs data before doing action. if there is error with filtering this will give us all datas

**Reply by kozakserwis** [reply]
The Android app doesn't work as well either. To be able to take measurements, I must set the meter to the resistance position, so that the display shows OL, otherwise if, for example, DC measurement is set and any indication appears on the meter, the application crashes. So I have to set the meter to the resistance position, connect to the meter, turn off the recording of the graph and then everything works, I can take measurements.

See on video: https://www.youtube.com/watch?v=vyYLEH7GC9o

**Reply by kozakserwis** [reply]
In the portable version there is a log 2022-11-20 but no new one is created

**Reply by webspiderteam** [reply]
i think original app also has problems with decoding. but no mather what we can find a way to fix our program. all I need is logs. 

> can you try this? [Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424586/Portable.zip)
> 
> this version logs data before doing action. if there is error with filtering this will give us all datas

if you try this version probably we can get logs. If not we should try different ways. But first try this one please

**Reply by kozakserwis** [reply]
Ok, now I have created a log in the version you sent me
[log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424777/log-2023-01-16.txt)

**Reply by webspiderteam** [reply]
> In the portable version there is a log 2022-11-20 but no new one is created

in portable the is no log. If you downloaded on release page I didnt mean it. 

download this file 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424809/Portable.zip)

**Reply by webspiderteam** [reply]
> Ok, now I have created a log in the version you sent me
> [log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424777/log-2023-01-16.txt)

ok now we got some data :) now we can do something :) which mode was it while logging?

**Reply by kozakserwis** [reply]
I dont rememer :) here is log under DC voltage
[log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10424873/log-2023-01-16.txt)

**Reply by webspiderteam** [reply]
ok now we are getting somewhere :) 

[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10425080/Portable.zip)
with this version you will see some measurements but there will errors. but its ok. if we can find which wrong displayed item is which we can fix. so you need to do stable measurements so we can fix error.

**Reply by kozakserwis** [reply]
I measured capacitor 1370uF
[log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10425234/log-2023-01-16.txt)
![Bez tytułu](https://user-images.githubusercontent.com/122780996/212665832-bf6610cf-90fb-4a17-8a37-c4030b0dc312.jpg)

On hold button is voltage measurment. I don't quite know what to do, the program shows y.000 or x.000, sometimes y.001 Should I measure capacitance, voltage and resistance and send a log file?

**Reply by kozakserwis** [reply]
https://www.youtube.com/watch?v=bQ6R_N7Hwnw

**Reply by webspiderteam** [reply]
we can fix mode button issues later because first we need to fix display first. There is a problem with digits. in original app gives us a different place but sample data is different.



> I measured capacitor 1370uF
> [log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10425234/log-2023-01-16.txt)

is that the only measurements for it?

**Reply by kozakserwis** [reply]
Measurements taken:
capacity
DC voltage
resistance on continuity test (there is no button for resistance)
Log in attachment
[log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10425523/log-2023-01-16.txt)
https://www.youtube.com/watch?v=5czgT9928ck

**Comment by webspiderteam** [comment]
can you check if digits are correct 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10425684/Portable.zip)

**Comment by kozakserwis** [comment]
Yes, now digits are correct but instead of 19.000V is ?9.000V. In restistance mode is ?08.30 Ohm  instead of 08.30 Ohm etc...

**Reply by webspiderteam** [reply]
Ok now we can fix other problems. İf you can record video like you did last time i can fix all quickly. But I need to see both displays clearly (multimeter and program). İf you can record close to them and send me to log it would probably fixed 90 percent. And please clear logs and start program after recording. So then close app before stop recording. This way I can realize which data is for which measurement. Btw thanks for helping and giving us to your time 🙏

**Reply by kozakserwis** [reply]
I thank you for your time. Sorry for my poor english :)
[log-2023-01-16.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10426060/log-2023-01-16.txt)
https://www.youtube.com/watch?v=CnS9jZkbxio

**Reply by webspiderteam** [reply]
Its ok. My english is also not good :) 

I will test this and then I will return as soon as posible ;)

**Reply by webspiderteam** [reply]
ok now most faults has to be fixed but I'm not sure yet. please check for errors.
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10426917/Portable.zip)

**Reply by ludwich66** [reply]
@Engin 
If the new adjustments affect the existing code, I'm happy to test them.
Also please let me know about the new mapping so I can update the Github pages.
Glad it's going again!!
Greetings Ludwich

**Reply by webspiderteam** [reply]
> If the new adjustments affect the existing code, I'm happy to test them.

I dont think it can effect old devices but testing is always good :)



> Also please let me know about the new mapping so I can update the Github pages.

I will but I'm not sure it is corrected now. it look like It look like I sliped bytes.

**Reply by kozakserwis** [reply]
Resistance and capacity seems to be ok, but DC voltage is not ok. Source 19.610V DC shows as 9:610 and after reversing the polarity, "minus" is not displayed
![Bez tytułu](https://user-images.githubusercontent.com/122780996/212718046-684d81c4-e6da-4800-8848-872ba40fad57.jpg)

**Reply by webspiderteam** [reply]
Can you send me data for 19.610 V and -19.610 V. There is a problem with first digit and minus area. I need data to fix it.

**Reply by kozakserwis** [reply]
Sure
[-19v.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10432752/-19v.txt)
[19V.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10432753/19V.txt)
[-10V DC.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10432843/-10V.DC.txt)
[+10V DC.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10432844/%2B10V.DC.txt)

AC mode looks like ok, but not showing frequency. if this could be fixed and improved icon assignment it would be perfect :)

**Reply by webspiderteam** [reply]
ok that should work as expected but ofcourse not sure yet. 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10432947/Portable.zip)



> AC mode looks like ok, but not showing frequency. if this could be fixed and improved icon assignment it would be perfect :)

Probably there is no frequency data sended from device because official app also has no view for it but if you send data for it I will try to find out

**Reply by webspiderteam** [reply]
> improved icon assignment

which icons do you mean?

**Reply by kozakserwis** [reply]
> Prawdopodobnie nie ma danych o częstotliwości wysyłanych z urządzenia, ponieważ oficjalna aplikacja również nie ma dla niego widoku, ale jeśli wyślesz do niego dane, spróbuję się dowiedzieć

Then there is nothing to worry about, there is an oscilloscope for that :)

The DC current function is working perfectly now, there is one more minor note when measuring resistance and capacitance. There is an extra zero at the beginning
![Bez tytułu](https://user-images.githubusercontent.com/122780996/212854379-f6f5832c-8d9c-4dad-9e5a-ef545ed83cb2.png)
Can you remove it?

**Reply by kozakserwis** [reply]
> > improved icon assignment
> 
> which icons do you mean?


![Bez tytułu](https://user-images.githubusercontent.com/122780996/212857225-437de085-4700-4dfc-8fa2-0999e48542cd.png)

AUTO - do nothing
There is no resistance button

**Reply by webspiderteam** [reply]
i did but its a bug. so when I removed it, its displaying as -.0051 e.g like on multimeter 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10433271/Portable.zip)

Now its time to make long test for finding unseen errors. If there is not an other error I will try to fix mode button.

**Reply by kozakserwis** [reply]
> zrobiłem ale to błąd więc kiedy go usunąłem, wyświetlał się jako -.0051, np. jak na multimetrze [Portable.zip ](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10433271/Portable.zip)
> 
> Teraz czas na długi test na znalezienie niewidocznych błędów. Jeśli nie ma innego błędu, spróbuję naprawić przycisk trybu.

At the moment everything seems to be working fine. I will test the application and if I notice any bug, I will write :)

**Reply by webspiderteam** [reply]
> ![Bez tytułu](https://user-images.githubusercontent.com/122780996/212857225-437de085-4700-4dfc-8fa2-0999e48542cd.png)
> 
> AUTO - do nothing There is no resistance button

We are trying to find out to mode buttons datas but every model has different data for changing. I'm working on multimeter emulator to findout every mode button datas but it will take time. So you need to be patient for this. I will fix them as soon as posible

**Reply by webspiderteam** [reply]
> At the moment everything seems to be working fine. I will test the application and if I notice any bug, I will write :)

thanks a lot for helping. we were waiting someone to test this devices for mounts. but we couldn't find until now. So you make us so happy. Thank you so much :)

**Reply by kozakserwis** [reply]
This is a less important issue. The most important thing is that the program works correctly, changing modes is usually done with a button on the device anyway. AC/DC, resistance, capacitance, diode test, current measurement results are displayed correctly. Great job, thank you very much for this great app :)

**Reply by webspiderteam** [reply]
Yes you are right 😉 I am happy to know it's working as expected 😊 you're welcome ☺️ it's good to see it's helping people and they are using it and also they are happy 🙏 when I see our app on a video or somewhere on net that makes me feel happy. I will wait for your approval for releasing this version to public

**Reply by kozakserwis** [reply]
I think that soon I will record a video of the repair using your application :) Go ahead and publish this version :)

**Reply by ludwich66** [reply]
@kozakserwis
Po zintegrowaniu urządzenia możesz ponownie nagrać film. Należy zmierzyć jak najwięcej różnych wartości. (wolty AC/DC, ampery, omy, temperatura, częstotliwość...)
Następnie możemy wyciąć z niego małe przykładowe filmy.
To dużo pracy, ale „Jeden obraz jest wart tysiąca słów” (przysłowie)
Ludwich

+++

**Reply by kozakserwis** [reply]
Of course I will. P.S. ZT-5566SE does not measure temperature :)

**Reply by ludwich66** [reply]
TEMPERATURA: Czy w pokrętle nie ma wyświetlacza?
<img width="46" alt="image" src="https://user-images.githubusercontent.com/12202733/212866851-37ada3c8-ef09-44d0-8554-c991ffe2174a.png">

Czyli nie ma zewnętrznego czujnika?

L

**Reply by kozakserwis** [reply]
> TEMPERATURA: Czy w pokrętle nie ma wyświetlacza? <img alt="image" width="46" src="https://user-images.githubusercontent.com/12202733/212866851-37ada3c8-ef09-44d0-8554-c991ffe2174a.png">
> 
> Czyli nie ma zewnętrznego czujnika?
> 
> L

It only shows the temperature of the environment :) I have separate devices to measure the temperature

**Reply by ludwich66** [reply]
OK, Thanks ;-)

**Reply by webspiderteam** [reply]
> I think that soon I will record a video of the repair using your application :)

Made me happy to hear that :) 


> Go ahead and publish this version :)

I want to be sure before publishing. after long test result I will publish :) Btw @ludwich66 It would be great to test for any unexpected sideeffect :)

**Reply by ludwich66** [reply]
L: Please add your Last Testversion ;-)

**Reply by webspiderteam** [reply]
> L: Please add your Last Testversion ;-)

[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10433835/Portable.zip)
this one is the latest for now :)

**Reply by kozakserwis** [reply]
Short unpublic test video:
https://www.youtube.com/watch?v=A0ihGrW-CHw

You can add official version without red sing

1:27 - you can see how the s-ls-84 service power supply temporarily gives 15V when increasing the voltage. This is not a meter error, but a power supply error, so you have to be careful with it :) I use a better power supply to repair laptops :)

**Reply by webspiderteam** [reply]
Looks like it works fine 😊 I found an bug but not so important and not about this device. Analog meter stucks last position when displays O.L. I should fix it. But I didn't see any wrong display ☺️ that's good 😊 so I will upload new release on Saturday. Because I will wait if there's an unseen bug. Also I will work on emulator while waiting 😉 so maybe I can finish it and also can find mode button data's 🙄

**Reply by webspiderteam** [reply]
@kozakserwis  btw you should also test REL , HOLD, Min/Max and range functions over multimeter there can be a bug ;)
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10434599/Portable.zip)


> Analog meter stucks last position when displays O.L. I should fix it.

I fixed it but @ludwich66 you should test it because this section can effect csv export and mqtt data also can cause graph.

**Reply by ludwich66** [reply]
Ok, i do another Test - Spezial view at Analogmeter and Export.

**Reply by webspiderteam** [reply]
> Another Youtube Tester:
> [Kiss Analog, ZT-5566SE](https://www.youtube.com/watch?v=Er6hrOnAJRM)

ohh thats nice :) Can you contact with him. I don't know why but youtube deletes my mesages especially when I send link or email. Can you try it for this time?

**Reply by ludwich66** [reply]
Yes, i will do it after the Release this Version.

**Reply by webspiderteam** [reply]
> Ok, i do another Test - Spezial view at Analogmeter and Export.

Thanks alot :) You don't need to be in a hurry. As I said I will publish it on saturday ;)

**Reply by webspiderteam** [reply]
> Yes, i will do it after the Release this Version.

Ok. I made a beta release for this https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/releases/tag/WPFReleaseBeta1.24

**Reply by kozakserwis** [reply]
> @kozakserwis btw you should also test REL , HOLD, Min/Max and range functions over multimeter there can be a bug ;)

I tested it before. Everything seems to be ok

**Reply by webspiderteam** [reply]
> I tested it before. Everything is ok

Is app displays them correct? I it is then there shouldn't be any bug then I can release without doubt :)

**Reply by kozakserwis** [reply]
I checked again more thoroughly. HOLD function doesn't work, MAX/MIN and REL is ok :)

https://www.youtube.com/watch?v=Fx2WexE54Pc

**Reply by webspiderteam** [reply]
> I checked again more thoroughly. HOLD function doesn't work, MAX/MIN and REL is ok :)

actually hold is not working because device hold its own display but sending realtime data to app. I don't know if official app has same problem but its not about the app. also rel function is working but rel symbol is not showing on app the triangle symbol is for rel.

**Reply by webspiderteam** [reply]
this is getting hold but will not hold because of device sending data and also I tryed to figure out where the rel is. I wish I found it. If not I need data before rel and rel pressed. also removed contact to us strip :)
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10436468/Portable.zip)

**Reply by kozakserwis** [reply]
> to się trzyma, ale nie będzie się utrzymywać z powodu wysyłania danych przez urządzenie, a także próbowałem dowiedzieć się, gdzie jest rel. Chciałbym to znaleźć. Jeśli nie, potrzebuję danych przed naciśnięciem rel i rel. usunięto również pasek kontaktowy do nas :) [Portable.zip ](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10436468/Portable.zip)

[afret REL.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10438186/afret.REL.txt)
[before REL.txt](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10438187/before.REL.txt)

**Reply by webspiderteam** [reply]
My guess was right :) It should work with last one :) you will see triangle next to AC/DC when its in rel ;)

**Comment by ludwich66** [comment]
Youtube Problems!
I made a Comment under KissAnalog#s Video, Youtube deleted this Post. 
I think it is a Security function, if the find links in the Post.
I wrote a E-Mail to the Business from KissAnalog, i hope it is not a Problem...

L

**Reply by webspiderteam** [reply]
> I think it is a Security function, if the find links in the Post.

Probably... 
Ok then. We should wait for response. But I don't think anything changes. We already tested with @kozakserwis %95 of data is working. Only rel is missing. And hold has a bug that device sending realtime data. But we need to find mode buttons every models has different response so we need to find out each model.

**Reply by kozakserwis** [reply]
I sent the logs before pressing REL and after pressing REL. If I can help in any way, please write to me :)

**Reply by webspiderteam** [reply]
@kozakserwis Thank you so much :) there is nothing for now :) if there will be any data we need I will ask for it thanks again :)

**Reply by webspiderteam** [reply]
@ludwich66 the last shape of datas for 5566se is here
![image](https://user-images.githubusercontent.com/85828505/212987493-dde289d9-f07b-4820-aeaf-93ba8a822dc1.png)

![image](https://user-images.githubusercontent.com/85828505/212987652-21ccc09c-6eee-4c2f-9296-ca96277eff14.png)

![image](https://user-images.githubusercontent.com/85828505/212987968-7e9f1ab6-e376-477d-9432-4bde5af90f30.png)

**Reply by ludwich66** [reply]
Danke, ich werde es erst morgen ergänzen - ich melde mich bei Rückfragen.

Schönen Abend

Tolles Ergebnis!!

**Reply by webspiderteam** [reply]
> ![image](https://user-images.githubusercontent.com/85828505/212987652-21ccc09c-6eee-4c2f-9296-ca96277eff14.png)

ohh sorry this part was changed as below

![image](https://user-images.githubusercontent.com/85828505/212989361-49169631-ee30-4bfb-9003-000ec540b53f.png)

**Reply by webspiderteam** [reply]
Its ok :) whenever you want :)

**Comment by ludwich66** [comment]
Hello Engine,
can you please look over it again, if necessary just send me your complete table as an original file (libreoffice).
Greetings Ludwich

[BT-DMM_BITMASTER.ods](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10444070/BT-DMM_BITMASTER.ods)

**Reply by webspiderteam** [reply]
[BT-DMM_BITMASTER.ods](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10449354/BT-DMM_BITMASTER.ods)
only fixed last part

**Comment by ludwich66** [comment]
A New YT Video
Bob. Kaplan
Zotek ZT-5566ES talking multimeter 4.1/2 digits 20000 counts battery operated bench DMM
https://www.youtube.com/watch?v=KQFzXrznSYs
L

**Comment by webspiderteam** [comment]
@ludwich66 I finally got all mode datas 

Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 77 46 9A CB 32 71 67 7B  5566se
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 77 46 9A CB 32 71 67 7B starts  sends when connected but only for 5566se
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 ED A2 C1 32 71 64 99
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 ED A2 C1 32 71 64 99 auto
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E3 A2 C1 32 71 64 9B
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E3 A2 C1 32 71 64 9B c/f
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E5 A2 C1 32 71 64 81
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E5 A2 C1 32 71 64 81 cap
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E4 A2 C1 32 71 64 86
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E4 A2 C1 32 71 64 86 diode
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E7 A2 C1 32 71 64 87
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E7 A2 C1 32 71 64 87 ncv
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E6 A2 C1 32 71 64 84
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E6 A2 C1 32 71 64 84 hz
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 9C A2 C1 32 71 64 EE
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 9C A2 C1 32 71 64 EE a/ma
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E1 A2 C1 32 71 64 85
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 E1 A2 C1 32 71 64 85 hold
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 84 A2 C1 32 71 64 E6
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 84 A2 C1 32 71 64 E6 max/min
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 91 A2 C1 32 71 64 95 
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 91 A2 C1 32 71 64 95 v ac/dc
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 93 A2 C1 32 71 64 EB 
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 93 A2 C1 32 71 64 EB mvhz
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 EB A2 C1 32 71 64 93
Characteristic with Guid: 0000fff4-0000-1000-8000-00805f9b34fb changed: EA EC 70 EB A2 C1 32 71 64 93 ohm

zero is not sending anything or maybe sebding from somewhere else or only in some mode but I tryed various data and no response for that.

**Comment by webspiderteam** [comment]
@ludwich66 @kozakserwis Hey guys :) It's time to test mode buttons :) 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10468862/Portable.zip)

**Reply by ludwich66** [reply]
V05B:
Auto: OK
°C: OK
**°F: no, did it before** Position is exact over Hamburger Menue
Cap: OK
Diode: OK
NCF: OK
HZ: OK
HOLD: OK (Red Font)
Other not on V05B

Ludwich

**Reply by ludwich66** [reply]
How did you Hack this now?

**Reply by webspiderteam** [reply]
> °F: no, did it before
 
ok fixed

> Position is exact over Hamburger Menue

I know but there is 2 option for it 1. seperate them two line or 2. make them smaller 
both of them can be looks weird. I'll think about it.



> HZ:

there is no  hz measurement for your device. right?

**Reply by webspiderteam** [reply]
> How did you Hack this now?

not hacked actually. I made my PC a fake device and emuleted the dmm then connected to pc with official app :))

**Reply by ludwich66** [reply]
Please, send the fixed Version.

The position isn't a problem, I thought the windows got in the way :-)

Left HZ Button works fine on V05B.

+++

**Reply by webspiderteam** [reply]
> The position isn't a problem, I thought the windows got in the way :-)

what you mean? do you mean I can left it as it is?

**Reply by ludwich66** [reply]
Yes - let it be  :-)

**Comment by webspiderteam** [comment]
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10469084/Portable.zip)
here it is :) fixed version ;)

**Reply by ludwich66** [reply]
Great !!
Remote It is complete on V05B.!!!

-> ?? Disable unused Buttons (minMax, mVHz.  Ohm, A/mA , ACDC) if they are not supported on spec device ??

L

**Reply by webspiderteam** [reply]
but I dont know which butons working on which device. 

> Disable unused Buttons (minMax, mVHz. Ohm, A/mA , ACDC)

these buttons isnt working for you? did you ever try them with official app?

**Reply by webspiderteam** [reply]
> but I dont know which butons working on which device.
> 
> > Disable unused Buttons (minMax, mVHz. Ohm, A/mA , ACDC)
> 
> these buttons isnt working for you? did you ever try them with official app?

some buttons can usable on only some modes!

**Reply by ludwich66** [reply]
I think it was different even in the original software, with one all the buttons were there (like ours) with the other only the usable ones were there.
So no problem - for me it's OK :-)

**Reply by webspiderteam** [reply]
> I think it was different even in the original software, with one all the buttons were there (like ours) with the other only the usable ones were there.

actually its all there If I remind correct. I saw only one change in buttons and its auto button. changed like that 
![Screenshot_2023-01-20-20-07-52](https://user-images.githubusercontent.com/85828505/213781661-6024cc84-0c42-4f85-8a9d-9f1004feb0cd.jpg)

**Comment by ludwich66** [comment]
I found in iPhone App a Button "ZERO" it works in Cap Mode to set Farad to "0000" first.

**Reply by webspiderteam** [reply]
thats nice but I dont have any IOS device :(

**Reply by ludwich66** [reply]
On your Android Picture is zero too - usable if you selectet Cap-Mode before??

**Reply by webspiderteam** [reply]
Then first I need to find cap mode first then I can try

**Reply by ludwich66** [reply]
Now we have communication problems again?
Cap mode --> measure farads then ZERO Button to Reset Farads to 0000?
or what did you meen?

What is  EA EC 70 E3 A2 C1 32 71 64 9B c/f on your Listing ??

**Reply by ludwich66** [reply]
Old Entry
<img width="617" alt="image" src="https://user-images.githubusercontent.com/12202733/213812813-17bb75d0-f2be-429e-94fa-6b8975084974.png">

**Reply by webspiderteam** [reply]
I mean my emulator app sending data to e-bull app that acts as multimeter. So my emulator have to send data as multimeter is in cap mode. That means I have to find which data was in capacitance measurement

**Reply by webspiderteam** [reply]
hahaha :) I found it :)) EA EC 70 E0 A2 C1 32 71 64 9A for zero in cap mode :)

**Reply by ludwich66** [reply]
Great, I was about to write that it's a relatively rarely required action. But having is better than needing ;-)
L

**Comment by webspiderteam** [comment]
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10470689/Portable.zip)
zero added :)

**Reply by webspiderteam** [reply]
@ludwich66 can you try this too.
https://github.com/webspiderteam/Bluetooth_DMM_For_Windows_MQTT_Test_Project/releases/tag/emulator1.0

this is the emulator app. but before use it you have to make some tweak.

**Reply by ludwich66** [reply]
Morning Engin,
good idea to split buttons!!
But Zero dosn't work for v05B (Check: IPhone App does it)
Next to do view Emulator!
L

**Reply by webspiderteam** [reply]
sorry about that. its typo mistake. 
[Portable.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/10472039/Portable.zip)

> Next to do view Emulator!

you have to change your PC's bluetooth name to Bluetooth DMM otherwise app does not see ;)

**Comment by webspiderteam** [comment]
1.24 Released friends :) thanks for everything 🥇 I couldn't do it without your helps. Thanks alot. BTW I still want to findout if 5566se sends any data for Hz readings. @kozakserwis If you can send me data when you got free time it would be great... Thank you all again 👍
